### PR TITLE
BREAKING CHANGE: parking orbit

### PR DESCRIPTION
**Pull request by cristiand391**
### What does this PR do?
bump oclif/core, command, deps.

**BREAKING CHANGE**:
this version removes support for lerna monorepos.

### What issues does this PR fix or reference?
@W-11526654@

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -2,10 +2,16 @@
 
 const oclif = require('@oclif/core');
 
-require('ts-node').register();
```
Comment (mshanemc) [inline bin/dev:5]: should bin/run have also changed?

File: test/registry.test.ts [hunk 2]
```diff
@@ -42,21 +51,21 @@ describe('npmrc tests', () => {
...
   afterEach(() => {
-    fs.removeSync(packageDir);
+    fs.rmSync(packageDir, { recursive: true });
   });
   it('should NOT WRITE npmrc registry for registry defaults', async () => {
     $$.SANDBOX.stub(Env.prototype, 'getString').returns(undefined);
     const registry = new Registry();
     await registry.setNpmRegistry(packageDir);
-    expect(fs.fileExistsSync(path.join(packageDir, '.npmrc'))).to.be.false;
+    expect(fileExistsSync(path.join(packageDir, '.npmrc'))).to.be.false;
```
[the remark this answers is not in the document]
Comment (cristiand391) [reply test/registry.test.ts:63]: oh right, I thought it was deprecated: https://nodejs.org/api/fs.html#fsexistssyncpath
I'll use that then.

File: package.json [hunk 3]
```diff
@@ -1,31 +1,29 @@
...
   "bin": {
     "sf-release": "bin/run"
   },
   "dependencies": {
-    "@oclif/config": "^1",
-    "@octokit/core": "^3.4.0",
-    "@salesforce/command": "^4.2.0",
-    "@salesforce/core": "^2.31.0",
+    "@oclif/core": "^1.13.3",
+    "@octokit/core": "^4.0.4",
```
Comment (cristiand391) [inline package.json:12]: octokit breaking changes v3 -> v3:
https://github.com/octokit/core.js/releases/tag/v4.0.0

File: package.json [hunk 4]
```diff
@@ -65,7 +60,6 @@
     "eslint-plugin-jsdoc": "^35.1.2",
     "eslint-plugin-prettier": "^3.1.3",
     "husky": "^7.0.4",
-    "lint-staged": "^11.2.3",
```
Comment (cristiand391) [inline package.json:68]: see [W-10957016](https://gus.my.salesforce.com/a07EE00000uNyEXYA0)

**Review by mshanemc — APPROVED** [review]

**Comment by mshanemc** [comment]
QA notes
I'm not gonna check them all, just a spot check of ones that call others (ex: latest-rc build calls a lot of other tasks in this plugin)

`sfdx cli:releasenotes`
✅ works with both CLIs

`dependabot:automerge` 
✅ worked on STL https://github.com/forcedotcom/source-tracking/pull/194 (using my gh token)

`latest-rc:build`
✅ https://github.com/salesforcecli/sfdx-cli/pull/641/files seems right.  Did bump SDR, no other plugins changed

`plugins:trust:verify`
✅  verified plugin-auth

**Pull request merged**